### PR TITLE
Larger sidedocs in Monaco, shrink font & no wrap for python

### PR DESCRIPTION
**Pull request by shakao**
image

Some fixes for https://github.com/microsoft/pxt-minecraft/issues/1521

Sidedocs take up 50% of the screen in text (py/js), unchanged in blocks

### CODE REVIEW

**Comment by pelikhan** [comment]
Looks like the function names are being incremented on each snippet. @darzu

**Comment by shakao** [comment]
oh no this was just a fake/local tutorial where i pasted in the converted python code so i could test

**Comment by darzu** [comment]
Huge improvement, love it!

File: theme/docs.less [hunk 1]
```diff
@@ -472,3 +472,14 @@
     }
 }
 .replaceDocsAvatar();
+
+#docs.py {
```
Comment (pelikhan) [inline theme/docs.less]: does this apply to javascript too?

File: webapp/public/docs.html [hunk 2]
```diff
@@ -119,6 +119,9 @@
             var backLabel = document.getElementById('back-label');
             if (backLabel)
                 backLabel.textContent = pxt.Util.lf("Go back")
+            var editorlang = pxt.Util.getEditorLanguagePref();
+            if (editorlang == "py") document.body.classList.add("py");
```
Comment (pelikhan) [inline webapp/public/docs.html]: IMO, allways add the editorlang to the class list.

File: theme/docs.less (hunk 1 above)
Comment (shakao) [reply theme/docs.less]: yeah you're right, i think it should. updated to use a single editorlang-text class to style both js/python

**Pull request merged**